probeinterface/io.py

```diff
@@ -754,7 +754,9 @@ def read_mearec(file):
     # set device indices
     if elinfo["sortlist"][()] != "null":
         channel_indices = elinfo["sortlist"][()]
-        probe.set_device_channel_indices(channel_indices)
+    else:
+        channel_indices = np.arange(positions.shape[0], dtype='int64')
```

Comment:
Isn't this default?

Reply:
No by default a probe is "unwired" (None).
In your case, you want to wire to traces channels always.